docs/src/resampling.md

```diff
@@ -67,10 +67,16 @@ The right image is the upsampled version of the left one.
 ![](assets/image_low_res.png)
 ![](assets/image_high_res.png)
 
+There are fast versions (`upsample2`) for upsampling by a factor of 2.
 
 
 ## Function References
 ```@docs
 FourierTools.resample
+FourierTools.resample_by_FFT
+FourierTools.resample_by_RFFT
+FourierTools.resample_nfft
+FourierTools.resample_czt
```

Comment:
I would remove all of them.

Reply:
why? These are useful methods. And it would break backward compatibiltiy.

Reply:
Then you messed your git. 
Indeed it was exported before (not so nice tbh). But why did you add the FourierTools. module before them?
Look here for status quo: https://github.com/bionanoimaging/FourierTools.jl/blame/87352d2e0844a99b29693823d28b836ee654ed46/src/resampling.jl#L8

Reply:
I don't understand your question. Can you explain?

Reply:
Ah, that's docs. my fault.